deploy/olm-catalog/bundle/manifests/konveyor-operator.v99.0.0.clusterserviceversion.yaml

```diff
@@ -759,48 +759,48 @@ spec:
                 - name: HOOK_RUNNER_REPO
                   value: hook-runner
                 - name: HOOK_RUNNER_TAG
-                  value: latest
+                  value: release-1.3.2
                 - name: MIG_CONTROLLER_REPO
                   value: mig-controller
                 - name: MIG_CONTROLLER_TAG
-                  value: latest
+                  value: release-1.3.2
                 - name: MIG_UI_REPO
                   value: mig-ui
                 - name: MIG_UI_TAG
-                  value: latest
+                  value: release-1.3.2
                 - name: MIGRATION_REGISTRY_REPO
                   value: registry
                 - name: MIGRATION_REGISTRY_TAG
-                  value: latest
+                  value: release_2.7
                 - name: VELERO_REPO
                   value: velero
                 - name: VELERO_TAG
-                  value: latest
+                  value: konveyor-1.4.2
                 - name: VELERO_PLUGIN_REPO
                   value: openshift-velero-plugin
                 - name: VELERO_PLUGIN_TAG
-                  value: latest
+                  value: release-1.3.2
                 - name: VELERO_AWS_PLUGIN_REPO
                   value: velero-plugin-for-aws
                 - name: VELERO_AWS_PLUGIN_TAG
-                  value: latest
+                  value: konveyor-1.1.0
                 - name: VELERO_GCP_PLUGIN_REPO
                   value: velero-plugin-for-gcp
                 - name: VELERO_GCP_PLUGIN_TAG
-                  value: latest
+                  value: konveyor-1.1.0
                 - name: VELERO_AZURE_PLUGIN_REPO
                   value: velero-plugin-for-microsoft-azure
                 - name: VELERO_AZURE_PLUGIN_TAG
-                  value: latest
+                  value: konveyor-1.1.0
                 - name: VELERO_RESTIC_RESTORE_HELPER_REPO
                   value: velero-restic-restore-helper
                 - name: VELERO_RESTIC_RESTORE_HELPER_TAG
-                  value: latest
+                  value: konveyor-1.4.2
```

Comment:
Address CVE-2020-3996 in `konveyor-1.4.3` ([1.4.3](https://github.com/konveyor/velero/tree/konveyor-1.4.3))